Base/usr/share/man/man4/audio.md

```diff
@@ -11,7 +11,7 @@ The `/dev/audio` character device file exposes the audio output device of the sy
 | Format | 16-bit signed | 16-bit signed |
 | Data | Left sample | Right sample |
 
-The sample rate of the samples is determined by the audio device's current sample rate, which may be accessed by an [ioctl](../man3/ioctl.md).
+The sample rate of the samples is determined by the audio device's current sample rate, which may be accessed by an [ioctl](../man2/ioctl.md).
```

Comment:
Looks like you already fixed the bug you wanted to check for here?

Reply:
I know, I'm currently trying to get Azure to work :P